app/experimenter/templates/experiments/section_overview.html

```diff
@@ -24,9 +24,6 @@
     <p>{{ experiment.engineering_owner }}</p>
   {% endif %}
 
-  <p><strong>Related Work</strong></p>
-  {{ experiment.related_work|urlize|linebreaks }}
-
   {% if experiment.related_to.all %}
```

Comment:
Do we need to remove this from the form as well?

Reply:
right right